### PR TITLE
atabek - week1

### PR DESCRIPTION
**Pull request by atabek-bakhramov**

### CODE REVIEW

File: Week1/homework/exercises/ex2.js [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+  // What are the names of all the cities in the Netherlands?
+  "SELECT name FROM city WHERE countryCode = 'NLD'",
+  // What is the population of Rotterdam?
+  "SELECT population FROM city WHERE name = 'Rotterdam'",
+  // What's the top 10 countries by Surface Area?
+  'SELECT name, surfaceArea From country ORDER BY surfaceArea DESC LIMIT 10',
+  // What's the top 10 most populated cities?
+  'SELECT name, population From city ORDER BY population DESC LIMIT 10',
+  // What is the population number of the world?
+  'SELECT SUM(population) From country',
```
Comment (GinaSt) [inline Week1/homework/exercises/ex2.js:31]: Here you could give a name to your resulting column: SELECT SUM(population) as WorldPopulation From country;

File: Week1/homework/exercises/ex1.js [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+connection.query('CREATE DATABASE meetup', error => {
+  if (error) throw error;
+  console.log('Meetup database is created.');
+});
+
+connection.query('USE meetup', error => {
+  if (error) throw error;
+});
+
+const tableInvitee = 'CREATE TABLE Invitee (invitee_no INT, invitee_name VARCHAR(50), invited_by VARCHAR(50))';
```
Comment (GinaSt) [inline Week1/homework/exercises/ex1.js:24]: Primary keys are missing from your tables.

File: Week1/homework/exercises/ex1.js [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+});
+
+connection.connect();
+
+connection.query('DROP DATABASE IF EXISTS meetup', error => {
+  if (error) throw error;
+  console.log('Database is dropped.');
+});
+
+connection.query('CREATE DATABASE meetup', error => {
```
Comment (GinaSt) [inline Week1/homework/exercises/ex1.js:15]: You should create it "if it doesn't exist"!

File: Week1/homework/exercises/ex1.js [hunk 4]
```diff
@@ -0,0 +1,75 @@
...
+  "INSERT INTO Invitee VALUES (1, 'Yoyo', 'Ata')",
+  "INSERT INTO Invitee VALUES (2, 'Baba', 'Bono')",
+  "INSERT INTO Invitee VALUES (3, 'Duda', 'Bono')",
+  "INSERT INTO Invitee VALUES (4, 'Mama', 'Ata')",
+  "INSERT INTO Invitee VALUES (5, 'Popo', 'Bono')",
+];
+
+const roomTableValues = [
+  "INSERT INTO Room VALUES (11, 'Conference Room', 1)",
+  "INSERT INTO Room VALUES (11, 'Conference Room', 1)",
```
Comment (GinaSt) [inline Week1/homework/exercises/ex1.js:45]: What is the purpose of storing multiple times the same information? The table "Rooms" should contain one line per room. Here you have five lines for only 2 rooms. The same goes for your meetings table (the rows of the table should represent different meetings). If you had added Primary keys to your tables (which you should always do) these insertions would not have been allowed.

**Review by GinaSt — APPROVED** [review]
Hello!

Your work seems good! Keep in mind that it is important to always add primary keys to your table and make sure you have understood that each row in the tables is supposed to be unique (otherwise you are storing redundant information.)
I have added a few comments with suggestions :)

File: Week1/homework/exercises/ex2.js (hunk 1 above)
Comment (atabek-bakhramov) [reply Week1/homework/exercises/ex2.js:31]: Good tip!

File: Week1/homework/exercises/ex1.js (hunk 2 above)
Comment (atabek-bakhramov) [reply Week1/homework/exercises/ex1.js:24]: Thank you!

File: Week1/homework/exercises/ex1.js (hunk 3 above)
Comment (atabek-bakhramov) [reply Week1/homework/exercises/ex1.js:15]: Thank you, now I know :)

File: Week1/homework/exercises/ex1.js (hunk 4 above)
Comment (atabek-bakhramov) [reply Week1/homework/exercises/ex1.js:45]: Thank you! I see, my code doesn't make much sense.

**Comment by atabek-bakhramov** [comment]
> Hello!
> 
> Your work seems good! Keep in mind that it is important to always add primary keys to your table and make sure you have understood that each row in the tables is supposed to be unique (otherwise you are storing redundant information.)
> I have added a few comments with suggestions :)

Thank you very much @GinaSt for your feedback! Much appreciated!

File: Week1/homework/exercises/ex1.js (hunk 2 above)
Comment (GinaSt) [reply Week1/homework/exercises/ex1.js:24]: Ok then! No worries about it!

**Pull request closed without merging**